### DISCUSSION TITLE
Vercel Takeover no longer possible!!

### DISCUSSION THREAD

**Discussion by zangcc** [projectdiscovery/nuclei-templates#9985] [Q&A]
Is this vulnerability no longer exploitable? Why does nuclei still display this script?
<img width="944" alt="image" src="https://github.com/EdOverflow/can-i-take-over-xyz/assets/64825932/54beb2bf-1769-4ef1-8073-ba8f64e741ca">

![image](https://github.com/projectdiscovery/nuclei-templates/assets/64825932/786271b3-f93c-423c-8312-755894a59605)

**Comment by alicemara** [comment]
It is still possible but has some constraints. 

Good summary:
https://github.com/EdOverflow/can-i-take-over-xyz/pull/375

**Comment by pdelteil** [comment]
> Is this vulnerability no longer exploitable? Why does nuclei still display this script? <img alt="image" width="944" src="https://private-user-images.githubusercontent.com/64825932/337194480-54beb2bf-1769-4ef1-8073-ba8f64e741ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2Njg0NTIsIm5iZiI6MTcxODY2ODE1MiwicGF0aCI6Ii82NDgyNTkzMi8zMzcxOTQ0ODAtNTRiZWIyYmYtMTc2OS00ZWYxLTgwNzMtYmE4ZjY0ZTc0MWNhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE3VDIzNDkxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFmZWVhYjRjMTk2YWQxZDQ2NDM0NDk5YzkwNzYwMjgwMTBlN2FlYjk3OGEyM2U4MWE2MTU2OWQ0MGU5ZGMxNzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.df0TZUDIQfOnGAiJsqP2oGoQAa6B3h2ExDEDFcqpeyY">
> 
> ![image](https://private-user-images.githubusercontent.com/64825932/337197122-786271b3-f93c-423c-8312-755894a59605.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg2Njg0NTIsIm5iZiI6MTcxODY2ODE1MiwicGF0aCI6Ii82NDgyNTkzMi8zMzcxOTcxMjItNzg2MjcxYjMtZjkzYy00MjNjLTgzMTItNzU1ODk0YTU5NjA1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE3VDIzNDkxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYwMjA4NjBmNGZlYTBhNjBkNzEyNjYyY2VjOGVhYjVlMjVlNmI2NzExNDIzNDliNTRkMTgzZTkyMWUwMTQwODkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.R9csHZ-8OreInQ-pHCEyoVqxnNo98R2Bb3BLVCmhWO8)

Hello there, 

In my opinion all takeovers should be considered as `Edge Cases`. On the other side the likelihood of finding a valid subdomain takeover using a public template and a public bug bounty program are very low, not impossible but you need to have a very good automation or just being lucky.
